README.md

```diff
@@ -253,6 +251,20 @@ myFastAPIServer = Server(
 )
 ```
 
+### Spring Boot Admin Using Basic Authentication
+Pyctuator supports registration with Spring Boot Admin that requires basic authentications. The credentials are provided when initializing the Pyctuator instance as follows:
+```python
+Pyctuator(
+    app,
+    "Flask Pyctuator",
+    "http://localhost:5000",
+    f"http://localhost:5000/pyctuator",
+    f"http://spring-boot-admin:8082/instances",
+    registration_username="admin",
+    registration_password="1234",
```

Comment:
this promotes a bad practice of putting credentials in code. Please add a comment here specifying that you should never commit usernames and passwords to git.

Reply:
Agree, was thinking maybe assuming some env-variable, but I think its better the users decide on the correct method of getting the password.

Reply:
I'm changing the example touse `os.getenv("sba-password")` instead